src/pg/.gitignore

```diff
@@ -0,0 +1,6 @@
+regression.diffs
+regression.out
+results/
+crankshaft--dev.sql
+crankshaft--dev--current.sql
+crankshaft--current--dev.sql
```

Comment:
I also feel the virtualenv should not be handled in that makefile, but we need, e.g., the version information extracted from the .control to determine the name of the environment (and other things already computed here...). Considered doing it all at the top level but the particularities of pgxs complicate the matter... I'll give it another thought.